spec/openapi.yaml

```diff
@@ -195,16 +95,13 @@ paths:
                 type: object
                 properties:
                   profiles:
-                    $ref: "#/components/schemas/OctoProfiles"
+                    $ref: "#/components/schemas/Profiles"
         403:
           $ref: "#/components/responses/Unauthorized"
+
   /api/printer:
```

Comment:
Is this for use instead of /api/telemetry?

Reply:
> Is this for use instead of /api/telemetry?
https://github.com/prusa3d/Prusa-Connect-Local/pull/45/files#diff-dfce4d9d11a31be72739d767812e1b3435a06045c19be411a28b517d2c208001R127

> How about prusa errors codes? Should it be here?
https://github.com/prusa3d/Prusa-Connect-Local/pull/45/files#diff-dfce4d9d11a31be72739d767812e1b3435a06045c19be411a28b517d2c208001R153